src/Compilers/VisualBasic/Portable/Binding/BinderFactory.vb

```diff
@@ -387,8 +387,9 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                                  SyntaxKind.GetAccessorStatement,
                                  SyntaxKind.AddHandlerAccessorStatement,
                                  SyntaxKind.RemoveHandlerAccessorStatement,
-                                 SyntaxKind.RaiseEventAccessorStatement
-                                ' Default values are not valid (and not bound) for lambda parameters or property accessors
+                                 SyntaxKind.RaiseEventAccessorStatement,
+                                 SyntaxKind.OperatorStatement
```

Comment:
>SyntaxKind.OperatorStatement [](start = 33, length = 28)

It is not clear why are we getting here for the test scenario. We are not asking questions about default value. It looks like ```Public Overrides Function VisitParameter(node As ParameterSyntax) As Binder``` should be changed changed to request a binder for NodeUsage.ParameterDefaultValue only if we are after non-missing ```=``` token.